man/tmpfiles.d.xml

```diff
@@ -340,7 +345,17 @@ L     /tmp/foobar -    -    -     -   /dev/null</programlisting>
           parameter. Note that lines of this type do not influence the
           effect of <varname>r</varname> or <varname>R</varname>
           lines. Lines of this type accept shell-style globs in place
-          of normal path names.  </para></listitem>
+          of normal path names.  </para>
+
+          <para>If the age argument is specified in this line, the path
+          will still be subject to time based cleanup, but the actual
+          behavior depends on the type of the path: Similar to <varname>C</varname>,
```

Comment:
*s*imilar*ly* to C

Reply:
"similarly to" does not appear to be grammatically correct, as an adverb alone is enough to modify a sentence, which leaves no place for the following "to xxx".
"similar to something" together forms an adverbial, and should be correct here.